src/shed/_codemods.py

```diff
@@ -29,7 +29,9 @@ def inner(fn):
         @m.leave(matcher)
         def wrapped(self, original_node, updated_node):
             if not m.matches(updated_node, matcher):
-                return updated_node
+                # given that this doesn't trigger... can we delete it or do we need a new
+                # test case with one of the remaining codemods?
+                return updated_node  # pragma: no cover
```

Comment:
Needs a new test, due to https://github.com/Instagram/LibCST/issues/888.

(or for that to be fixed upstream and then shed requires the new version, but that's a lot of scope creep)

Reply:
I spent a while looking through the remaining codemods if any pair of them could trigger this, but didn't manage to find any. But I don't think it's much of a performance overhead so let's just leave it for now in case I missed something, or a new codemod is introduced that could trigger it.
Updated the docstring & comment